target/linux/ramips/rt305x/base-files/etc/board.d/01_leds

```diff
@@ -42,6 +42,9 @@ nexx,wt1520-4m|\
 nexx,wt1520-8m|\
 skyline,sl-r7205|\
 tenda,w306r-v2|\
+zyxel,keenetic-lite_rev.b)
+	set_wifi_led "rt2800soc-phy0::radio"
```

Comment:
This will break all devices above.
Despite, why does this deviate from keenetic-start?

Reply:
fixed

Reply:
So, what's with my question? Why have you chanded the trigger now?

Reply:
The trigger remains the same (set_wifi_led "rt2800soc-phy0 :: radio"). Based on https://github.com/trexxet/openwrt-zyxel_keenetic/blob/master/0019-ramips-add-ZyXel-Keenetic-userspace.patch
Working fine, I uploaded a new image after making changes.

Reply:
Could you check what `grep . /sys/class/leds/*/trigger` returns?

Reply:
`root@OpenWrt:~# grep .  /sys/class/leds/*/trigger`
`/sys/class/leds/keenetic-lite-b:green:internet/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt`
`/sys/class/leds/keenetic-lite-b:green:power/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt`
`/sys/class/leds/keenetic-lite-b:green:wlan/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt`
`/sys/class/leds/keenetic-lite-b:green:wps/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt`
`/sys/class/leds/rt2800soc-phy0::assoc/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt`
`/sys/class/leds/rt2800soc-phy0::quality/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt`
`/sys/class/leds/rt2800soc-phy0::radio/trigger:[none] timer default-on netdev heartbeat phy0rx phy0tx phy0assoc phy0radio phy0tpt
`

Reply:
So there's no `rt2800pci-phy0::radio` LED on your device, and none of your LEDs is using phy0* trigger. Does any LED turn on whey you enable wlan?

Reply:
I so sorry for this. My programmer skill is very low. Moved the router to the `set_wifi_led trigger "rt2800soc-phy0 :: radio"` trigger now. Uploaded new image, work fine.
Interestingly, with the `rt2800pci-phy0::radio` trigger, the wifi LED lights up when the wlan is turned on.

Reply:
If the wifi led uses WLAN_LED_N pin, then you don't need any LED setting, it will automatically turn on when you enable wifi.
You can turn on a LED by `echo 1 > /sys/class/leds/<LED>/brightness`
and turn it off by `echo 0 > /sys/class/leds/<LED>/brightness`.
Could you check if all LEDs turn on/off correctly with this command?

Reply:
work:
`echo 1 > /sys/class/leds/keenetic-lite-b:green:power/brightness`
`echo 1 > /sys/class/leds/keenetic-lite-b:green:wps/brightness`
`echo 1 > /sys/class/leds/rt2800soc-phy0::radio/brightness`

not work:
`echo 1 > /sys/class/leds/keenetic-lite-b:green:wlan/brightness`
`echo 1 > /sys/class/leds/keenetic-lite-b:green:internet/brightness`
`echo 1 > /sys/class/leds/rt2800soc-phy0::assoc/brightness`
`echo 1 > /sys/class/leds/rt2800soc-phy0::quality/brightness`

Reply:
So it seems wlan and internet LED do not exist, please drop the nodes in dts. And probably you won't need the 01_leds setting as well.

Reply:
Removed wlan and internet LEDs. Returned the `01_leds` file to its original state.